mesonbuild/interpreter/interpreter.py

```diff
@@ -140,6 +142,9 @@ def _project_version_validator(value: T.Union[T.List, str, mesonlib.File, None])
             return 'when passed as array must contain a File'
     return None
 
+def _get_meson_modules(module_path: Path) -> T.List[str]:
+    return [mod.name for mod in pkgutil.iter_modules(module_path) if not mod.name.startswith('_')]
+
```

Comment:
This function is only ever used in exactly one place, once. I would inline it (and import pkgutil at the inline call site in the error branch).